pkg/refactor/service/service.go

```diff
@@ -44,6 +44,11 @@ func getDockerServiceURL(namespace string, typ string, file string, name string)
 
 func SetupGetServiceURLFunc(config *core.Config, withDocker bool) {
 	GetServiceURL = func(namespace string, typ string, file string, name string) string {
+		// translate args for system-wide services.
```

Comment:
@alankm , I believe this is the only patch needed to enable system-wide services.

Reply:
Please, try it and let me know if I missed something.